DQMOffline/MuonDPG/interface/BaseTnPEfficiencyTask.h

```diff
@@ -0,0 +1,108 @@
+#ifndef BaseTnPEfficiencyTask_H
+#define BaseTnPEfficiencyTask_H
+
+/*
+ * \file BaseTnPEfficiencyTask.h
+ *
+ * \author L. Lunerti - INFN Bologna
+ *
+*/
+
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
```

Comment:
`ESHandle` doesn't look to be needed anywhere, if I am not wrong (and the same in all similar occurrencies in the derived classes)

Reply:
Removing it (see [previous comment](https://github.com/cms-sw/cmssw/pull/36276/files/d5cceeb5969684a11b082de131ebabbe621d4210#r784757551))